browser/ui/brave_browser_window.h

```diff
@@ -44,7 +44,13 @@ class BraveBrowserWindow : public BrowserWindow {
   // the overall screen's height
   virtual gfx::Rect GetShieldsBubbleRect();
 
-  virtual std::map<int, std::vector<ui::Accelerator>> GetAcceleratedCommands();
+  virtual base::flat_map<int, std::vector<ui::Accelerator>>
+  GetAcceleratedCommands();
+  virtual void AddAcceleratorForCommand(int command_id,
```

Comment:
optional: maybe AddOrUpdateAcceleratorForCommand() would be a better name for this

Reply:
Should we stick to uint32_t for |command_id| here too?

Reply:
Updated to using `int` everywhere - as the commands themselves are ints.
Renamed to `AssignAcceleratorToCommand` and `UnassignAcceleratorFromCommand`